java/client/src/main/java/glide/api/RedisClient.java

```diff
@@ -201,4 +203,144 @@ public CompletableFuture<String> set(String key, String value, SetOptions option
                         .build();
         return commandManager.submitNewCommand(command, BaseClient::handleStringResponse);
     }
+
+    /**
+     * Decrements the number stored at `key` by one. If `key` does not exist, it is set to 0 before
```

Comment:
```suggestion
     * Decrements the number stored at <em>key</em> by one. If <em>key</em> does not exist, it is set to 0 before
```
please repeat this for all functions